telemetry/aiwrapper.go

```diff
@@ -47,8 +47,10 @@ func SendAITelemetry(cnireport CNIReport) {
 	var msg string
 	if cnireport.ErrorMessage != "" {
 		msg = cnireport.ErrorMessage
-	} else {
+	} else if cnireport.EventMessage != "" {
 		msg = cnireport.EventMessage
+	} else {
+		return
 	}
 
 	report := aitelemetry.Report{
```

Comment:
can you also remove cni prefix from msg, no need to set prefix

line 57: Message:          "CNI:" + msg,